<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtSdlTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="SdlPovWidget" name="pov" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SdlAxisWidget" name="xyaxis" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="SdlAxisWidget" name="zaxis" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="buttonsLayout"/>
   </widget>
   <widget class="SdlSliderWidget" name="axis4" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>16</width>
      <height>128</height>
     </rect>
    </property>
   </widget>
   <widget class="SdlSliderWidget" name="axis5" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>16</width>
      <height>128</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="choosePadBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="testHaptic">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test haptic</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SdlAxisWidget</class>
   <extends>QWidget</extends>
   <header>sdlaxiswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SdlPovWidget</class>
   <extends>QWidget</extends>
   <header>sdlpovwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SdlSliderWidget</class>
   <extends>QWidget</extends>
   <header>sdlsliderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
